bin/recipes/001-import.js

```diff
@@ -67,14 +68,21 @@ async.parallel({
 
 
 		// imported recipes for search data
-		helper.create(options, Recipe2, (err, data) => {
+		helper.create(options, Recipes2, (err, data) => {
 			console.log(data);
 		});
 
-		options.predata = await Department.find({ where: {
-			name: 'Meat'
-		}});
-		console.log(options.predata);
+		const meatDepartments = Department.get().filter(department => {	
```

Comment:
Department.get() will query all departments from database? i mean there can be a lot of information. I think my way was more easy for db system.